com.unicornsonlsd.finamp.yaml

```diff
@@ -0,0 +1,298 @@
+app-id: com.unicornsonlsd.finamp
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: finamp
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0:ro
```

Comment:
does this really support pipewire ?

Reply:
MPV supports native PipeWire output, so yes :)